### DISCUSSION TITLE
It's kinda broken out of the box :/ 500 errors

### DISCUSSION THREAD

**Discussion by decryptedchaos** [modoboa/modoboa#3144] [Help]
It took hours of tinkering,  and going behind it and manually redoing things it should have done automatically just to get into the web ui and even then.  

Can't create domains  when i click submit on the domain creation page it throws a 500 error for the api  

![morelikedumboboa](https://github.com/modoboa/modoboa/assets/1302391/a51767a5-9cfc-463c-827b-9f9d3ef4472e)

**Comment by tonioo** [comment]
Can you enable debug traces? (set DEBUG to True inside settings.py and reload uwsgi)

**Reply by decryptedchaos** [reply]
I was able to get past that error by running a upgrade..  But it still has issues..  

It's "functional"  as I can create domains and users  but there are parts of the panel that don't work  graphs don't work  webmail throws a 500 error,   domains are forever pending DNS even though DNS is valid and working..
